tools/installer/requirements.txt

```diff
@@ -1 +1,2 @@
 mutagen==1.31
+ensurepip
```

Comment:
No. This should be the name of the module that's required. Probably should put pylast in here too, actually...